packages/block-editor/src/components/inserter/index.js

```diff
@@ -207,7 +207,7 @@ class Inserter extends Component {
 					'block-editor-inserter__popover',
 					{ 'is-quick': isQuick }
 				) }
-				position={ position }
+				popoverProps={ { position } }
```

Comment:
The switch to `placement` should happen in a separate PR (and requires exposing the `positionToPlacement` utility from `Popover` too)